### PR TITLE
added draw method to circuit

### PR DESCRIPTION
**Pull request by JamesB-1qbit**
Uses cirq's SVGCircuit such that the following code
```python3
from tangelo.linq import Circuit, Gate

circ = Circuit([Gate("CRZ", target=0, control=[1,2], parameter=1),
                Gate("CNOT", target=0, control=1),
                Gate("RZ", target=0, parameter=0.3)])
circ.draw()
```
makes
[output.svg.pdf](https://github.com/goodchemistryco/Tangelo/files/10427016/output.svg.pdf)

### CODE REVIEW

**Comment by ValentinS4t1qbit** [comment]
Thank you for this simple solution, James. Since Cirq is a requirement of Openfermion, itself a requirement of our package, I think this is a very reasonable default solution.

When I try to call the draw method in the basics notebook, I get a bunch of warning for a font not found, in my Ubuntu environment. I think 1 warning line per gate, which will quickly get unbearable. Is there any way we can silence those warnings or not having them come up in the first place ?

Screenshot from 2023-01-17 17-43-39

**Comment by JamesB-1qbit** [comment]
> 

I don't see these warnings and my virtual box seems to be broken. But I think that
```python3
warnings.filterwarnings("ignore", message=".*findfont.*")
```
I will commit this change and then someone who uses linux can test it. This error shows up on a bunch of cirq notebooks.

File: tangelo/linq/circuit.py [hunk 1]
```diff
@@ -144,6 +146,16 @@ def is_mixed_state(self):
         """
         return "MEASURE" in self.counts
 
+    def draw(self):
+        """Method to output a prettier version of the circuit for use in jupyter notebooks that uses cirq SVGCircuit"""
+        # circular import
+        from tangelo.linq.translator.translate_cirq import translate_c_to_cirq
+        warnings.filterwarnings("ignore", message=".*findfont.*")
```
Comment (ValentinS4t1qbit) [inline tangelo/linq/circuit.py:153]: This does not seem to work for me (I don't understand your pattern btw, I'm not sure why the dots are here).
Let's consider this a Cirq issue and remove this line as it does not work.

**Review by ValentinS4t1qbit — APPROVED** [review]
Thank you !

**Review by ValentinS4t1qbit — APPROVED** [review]

**Pull request merged**